src/integrationTest/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/BlobProcessorTaskTestForDisabledService.java

```diff
@@ -75,4 +81,36 @@ public void should_reject_file_when_service_is_disabled() throws Exception {
         errorWasSent(SAMPLE_ZIP_FILE_NAME, ErrorCode.ERR_SERVICE_DISABLED);
     }
 
+    @Test
+    public void should_not_move_file_if_send_message_throws_exception_when_service_is_disabled() throws Exception {
+        // given
+        byte[] zipBytes = zipDir("zipcontents/ok");
```

Comment:
Could we maybe use an invalid file instead of using the assumption that the service is disabled? The test code doesn't make the service disabled - it relies on some external condition. It would be best if the test itself could say "look, this is the bit that will cause the rejection".

Reply:
Created new test class BlobProcessorTaskTestForFailingNotification